src/assets/sprites/buttons.json

```diff
@@ -0,0 +1,188 @@
+{
```

Comment:
What kind of file format is this? Might be good to know if we need to use this again in the future

Reply:
this is just generated from texture pack